gradle/test-intellij.versions.toml

```diff
@@ -1,2 +1,3 @@
 [versions]
-versionList = "IIC-241.15989.150,IIC-223.8836.41"
+# https://jb.gg/intellij-platform-builds-list
+versionList = "IIC-243.21565.193,IIC-223.8836.41"
```

Comment:
When we test the plugin, we should periodically bump the versions in here to match the lower and upper bound of release versions we support.